de_DE/webapp/src/main/webapp/i18n/vivo_all_de_DE.properties

```diff
@@ -948,3 +948,8 @@ create_and_link_type_webpage=Webpage
 claim_publications_by=Importieren Sie Veröffentlichungen mittels
 claim_publications_by_doi=DOI
 claim_publications_by_pmid=PubMed ID
+
+# FirstAndLastNameValidator.java
+first_name_empty_msg = Sie müssen einen Wert in das Feld Vorname eingeben.
+last_name_empty_msg = Sie müssen einen Wert in das Feld Nachname eingeben.
+malformed_last_name_msg = Das Nachnamenfeld darf kein Komma enthalten. Bitte geben Sie den Vornamen in das Feld Vorname ein.
```

Comment:
"Das Feld Nachname darf kein Komma enthalten." instead of "Das Nachnamenfeld darf kein Komma enthalten."